.github/workflows/prerelease.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: "Install Docs Requirements"
         run: "pip install -r docs/requirements.txt"
       - name: "Build Documentation"
-        run: "mkdocs build --no-directory-urls"
+        run: "mkdocs build --no-directory-urls --strict"
```

Comment:
Ah, sorry if I was unclear on my previous comment. I was suggesting that we add `mkdocs build --strict` to `ci_pullrequest.yml` and `ci_integration.yml` so that it gets run as a CI check on pull requests and on commits to `main`/`develop`/`next`. The pre-release and release builds probably do *not* need the `--strict` option.

Reply:
Not necessarily unclear, I understood that. I added the commands to the tasks.py, which covers the ci_pullrequest.yml and ci_integration.yml since they call `invoke unittest` which will build the docs with the strict command.  I don't see any reason why it couldn't also be in the prerelease/release, which is why I added it here. I can remove if you think it will be  a problem.

Reply:
Will `invoke unittest` fail and abort early if `mkdocs build --strict` fails?

Reply:
Even if it does, it might make for clearer CI results if we have a standalone mkdocs check alongside the other linters, before getting to `invoke unittest`? Thoughts?

Reply:
I don't have a strong opinion on this, but the mkdocs build --strict will still build the docs, which if you add it to the linters, it will build the docs then. This is not a big problem, but we also need these docs for the unittest, so I need to add this step in for the unittest as well, so if you run `invoke tests` without the lint_only flag, it will run that mkdocs build twice for the Nautobot Docs and the Example Plugin.

Reply:
Looks like `mkdocs build` only takes about 5 seconds or so, so I'm ok with it running an extra time if it makes the results clearer.